actix-web/custom-tracing-subscriber/src/main.rs

```diff
@@ -0,0 +1,29 @@
+use actix_web::{get, web::ServiceConfig};
+use shuttle_actix_web::ShuttleActixWeb;
+use tracing_subscriber::filter::{EnvFilter, LevelFilter};
+
+#[tracing::instrument(name = "hello")]
+#[get("/")]
+async fn hello_world() -> &'static str {
+    let msg = "Hello World!";
+    tracing::info!("{msg}");
+    msg
+}
+
+#[shuttle_runtime::main]
+async fn actix_web() -> ShuttleActixWeb<impl FnOnce(&mut ServiceConfig) + Send + Clone + 'static> {
+    tracing_subscriber::fmt()
```

Comment:
Maybe a comment about the fact that we need to write to stdout for shuttle to record our logs, and so we use the fmt subscriber which has sane defaults for applications and writes to stdout (https://docs.rs/tracing-subscriber/latest/tracing_subscriber/fmt/index.html). What do you think?